tests/rptest/scale_tests/many_partitions_test.py

```diff
@@ -416,7 +416,7 @@ def _tiered_storage_warmup(self, scale, topic_name):
         # such that actual segment size is as close as possible to the desired
         # size. In default configuration we run with `log_segment_size_jitter_percent=5`
         # adjust the messages size to always be larger than that.
```

Comment:
btw if the above comment makes sense to anyone please let me know. I am struggling to parse it.

Reply:
i think it's trying to say that ideally a message would be sized as 1 byte and there would be no segment size jitter, so we could be as precise as we wanted in affecting the behavior. but in reality batches aren't 1 byte, there is segment size jitter, and there are even non-content batches so being precise is hard?

Reply:
but this bit:

> we need to size messages
>        # such that actual segment size is as close as possible to the desired
>        # size.

makes it sound like the original idea was such that one message fills the segment immediately which seems in contrast to why we did then do a `0.06 *` in the code?